tests/services/test_feed_manager.py

```
@@ -0,0 +1,84 @@
+        <head>
+            <title>Test OPML</title>
+        </head>
+        <body>
+            <outline text="Technology" title="Technology">
+                <outline text="InvalidFeed" title="InvalidFeed" type="rss" xmlUrl="http://invalid.url/" />
+            </outline>
+        </body>
+    </opml>"""
+
+@pytest.fixture
+def feed_manager():
```

Ruff (0.8.0)

Missing return type annotation for public function `feed_manager`

(ANN201)